services/worker/src/worker/job_runners/dataset/compatible_libraries.py

```diff
@@ -634,6 +634,8 @@ def get_compatible_libraries_for_webdataset(
     "parquet": "parquet",
     "imagefolder": "imagefolder",
     "audiofolder": "audiofolder",
+    "text": "text",
+    "arrow": "arrow",
```

Comment:
seems a bit error-prone to have to list them here

Reply:
you mean: extracting this to our constants.py file, for example?